### DISCUSSION TITLE
Operationalized Social Media and Linear Directional Mean Address Drug Overdose Crisis in America

### DISCUSSION THREAD

**Discussion by sahrk** [uber/h3#896] [H3 Spotlight]
[Operationalized Social Media and Linear Directional Mean Address.pdf](https://github.com/user-attachments/files/16501869/Operationalized.Social.Media.and.Linear.Directional.Mean.Address.pdf)

**Comment by sahrk** [comment]
@isaacbrodsky I see you put a `paper` label on these, but it doesn't look like I have permission to add labels.

**Reply by isaacbrodsky** [reply]
I think we need to add some permission on the repository for you to do that, I'll look into it

**Reply by sahrk** [reply]
> I think we need to add some permission on the repository for you to do that, I'll look into it

@isaacbrodsky I did see an invitation to collaborate and I'm pretty sure I accepted it last week (the link is not operative any more). But I just added a paper to this Discussion and as far as I can tell l am still not able to add a Label. Sorry if I'm missing something obvious...

**Reply by isaacbrodsky** [reply]
I added some additional permissions -- please try again now

**Reply by sahrk** [reply]
That did the trick, thanks!
